### ISSUE TITLE
Block not visible when in quiz on 4.1.3

### ISSUE THREAD

**Issue by SYoung3000** [marxjohnson/moodle-block_accessibility#151]
We've got a block in a side panel with the accessibility block - as an admin, that's visible on a quiz, as a student, it's not visible after you visit a quiz page - is this a known issue or are there any possible workarounds?

The block seems to largely function on M4, from what we can see, but this is a specific area where it seems broken
